package.json

```diff
@@ -26,7 +26,7 @@
     "@govuk-react/select": "^0.7.1",
     "@govuk-react/table": "^0.7.1",
     "agentkeepalive": "^4.1.3",
-    "applicationinsights": "^1.8.2",
+    "applicationinsights": "^1.8.7",
     "applicationinsights-native-metrics": "^0.0.5",
     "axios": "^0.19.2",
```

Comment:
tried upgrading axios too here but didn't work, `config.url` was blank when should have been populated...